src/bitmap/store/array_store/mod.rs

```diff
@@ -106,6 +106,15 @@ impl ArrayStore {
         (pos_end - pos_start) as u64
     }
 
+    pub fn remove_first(&mut self, n: usize) {
+        self.vec.rotate_left(n.into());
+        self.vec.truncate(self.vec.len() - n as usize);
+    }
+
+    pub fn remove_last(&mut self, n: usize) {
+        self.vec.truncate(self.vec.len() - n as usize);
```

Comment:
```suggestion
        self.vec.truncate(self.vec.len() - n);
```

Reply:
Thank you.
`u64` is now specified, so the now required.